main.js

```diff
@@ -0,0 +1,110 @@
+// Initial game board
+var towersBoard = {
+    board: [
+        [5, 4, 3, 2, 1], // Peg A
+        [],              // Peg B
+        []               // Peg C
+    ]
+};
+
+Array.prototype.myForEach = function(callback) {
+    for (let i = 0; i < this.length; i++) {
+        var element = this[i];
+        callback(element);
+    }
+};
+
+function displayBoard() {
+    towersBoard.board.myForEach(function(peg) {
+        console.log(peg);
+    });
+};
+
+function moveDisc(fromPeg, toPeg) {
+    if (towersBoard.board[fromPeg].length === 0) {
```

Comment:
to help with readability this check could have been a const
```suggestion
    const isPegEmpty = towersBoard.board[fromPeg].length === 0;
    if (isPegEmpty) {
```